connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -429,9 +431,13 @@ private void convertMessages(ConsumerRecords<byte[], byte[]> msgs) {
                     msg.offset(),
                     ConnectUtils.checkAndConvertTimestamp(msg.timestamp()),
                     msg.timestampType());
+            log.trace("{} applying transformations to message with key {}, value {}", this, msg.key(), msg.value());
```

Comment:
What about using `keyAndSchema.value()` and `valueAndSchema.value()` instead of the raw bytes, but also adding the topic, partition, and offset?